### PR TITLE
Added link and button for the Discord server

### PR DESCRIPTION
**Pull request by eXpl0it3r**
* Adjusted wording a bit
* Image with a link are now being slightly highlighted on hover

---

## Alternative

---

## Hover

---

## ToDo

* [x] French translation
* [x] Setup IRC-Discord bridge

---

What do you prefer?

### CODE REVIEW

**Comment by Foaly** [comment]
Nice! I like the first version. Looks cleaner and more like the rest of the sites style.

**Comment by eXpl0it3r** [comment]
@mantognini / @LaurentGomila needs a French translation 🙂

File: community-fr.php [hunk 1]
```diff
@@ -37,18 +37,30 @@
...
+        <a href="https://chat.sfml-dev.org"><img src="<?php echo image('community/irc.png') ?>" alt="IRC" /></a>
+    </div>
+</div>
+
+<div class="home-section">
+    <div class="stagger-column-thin">
+        <a href="https://discord.gg/gPBHzJ8"><img src="<?php echo image('community/discord.png') ?>" alt="Discord" /></a>
+    </div>
+    <div class="stagger-column-wide">
+        <div class="title">&raquo; <a href="https://discord.gg/gPBHzJ8" title="Join our Discord server now">Discord</a></div>
```
Comment (LaurentGomila) [inline community-fr.php]: "Rejoignez notre serveur Discord"

File: community-fr.php [hunk 2]
```diff
@@ -37,18 +37,30 @@
...
+        <p>
+            For friendly help, on- and off-topic discussions and easy media sharing join our Discord server. With the help of an IRC-Discord bridge everyone is connected with each other!
```
Comment (LaurentGomila) [inline community-fr.php]: Pour une aide amicale, des discussions sur SFML ou d'autres sujets, et partager facilement des medias, rejoignez notre serveur Discord. Avec l'aide d'un pont IRC-Discord, tout le monde se retrouve connecté ensemble !

**Comment by LaurentGomila** [comment]
> @mantognini / @LaurentGomila needs a French translation

Done. @mantognini don't hesitate to improve/correct if needed.

**Pull request merged**